build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
Actually wait, do you mean crossgen-ing CoreFX components during the non-Windows product build in the pipeline? I suppose that could work, but it'd be pretty involved - we'd have to start restoring CoreFX packages during the non-windows build, which we don't already do, then crossgen those, package them, and restore them during the test build. I'd prefer the approach I mentioned above, unless I interpreted this wrong